docs/dg/dev/architecture/architectural-convention.md

```diff
@@ -1567,29 +1583,26 @@ Injecting columns from `Product` to a `Url` `domain` by defining `spy_url.schema
                 └── [RouterName]Plugin.php
 ```
 
+`Providers` and `Routers` are used for bootstrapping the [Yves](#applications) application.
+- Controller Provider: registers [Yves](#applications) [Controllers](#controller) of a module and binds them to a path.
+- Router: resolves a path into an [Yves](#applications) [Controller and Action](#controller).
 
-**Description**
-
-`Providers` and `Routers` are used during the bootstrap of [Yves](#applications) application.
-- **Controller Provider** - Registers [Yves](#applications) [Controllers](#controller) of a module and binds them to a path.
-- **Router** - Resolves a path into a [Yves](#applications) [Controller and Action](#controller).
-
-Controllers in [Zed](#zed) application layer are autowired and do not require manual registration. An `ExampleController::indexAction()` in `ExampleModule` can be accessed via `/example-module/example/index` URI.
+Controllers in the [Zed](#zed) application layer are autowired and don't require a manual registration. An `ExampleController::indexAction()` in `ExampleModule` can be accessed via the `/example-module/example/index` URI.
 
-**Conventions**
+#### Conventions
 - A `Controller Provider` needs to extend `\SprykerShop\Yves\ShopApplication\Plugin\Provider\AbstractYvesControllerProvider`.
 - A `Router` needs to extend `\SprykerShop\Yves\ShopRouter\Plugin\Router\AbstractRouter`.
-- A `providers` are `routers` are classified as a [Plugin](#plugin) and so the plugin's conventions apply.
```

Comment:
Something is not right about this sentence

Reply:
- `Providers` and `routers` are classified as a [Plugin](#plugin) and so the plugin's conventions apply.